ui.frontend/src/view/FormFieldBase.js

```diff
@@ -221,9 +222,72 @@ export default class FormFieldBase extends FormField {
     updateErrorMessage(errorMessage, state) {
         if (this.errorDiv) {
           this.errorDiv.innerHTML = state.errorMessage;
-          if (state.valid === false && !state.errorMessage) {
-            this.errorDiv.innerHTML = 'There is an error in the field';
-          }
+            if (state.valid === false && !state.errorMessage) {
+                let validationMessage = LanguageUtils.getTranslatedString(this.formContainer.getModel()._jsonModel.lang, "DefaultError");
```

Comment:
We have expose .lang so please use it directly `this.formContainer.getModel().lang`. Also, we should return a function, to make this more readable. Please check this, https://github.com/adobe/aem-core-forms-components/blob/master/ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/af-commons/v1/clientlibs/editor/utils/utils.js#L104-L124

Reply:
Usage would be the following,
`
let getTranslatedString = LanguageUtils.getTranslatedString(this.formContainer.getModel().lang);
// create constants for all validationState
// please make the code more readable
if (ValidationStateConstants.VALUE_MISSING in validationState) {
    getTranslatedString(`${ValidationStateConstants.VALUE_MISSING}DefaultError`)
}
`